src/server/websocket.coffee

```diff
@@ -11,6 +11,7 @@ wrapSession = (conn) ->
   wrapper.send = (response) ->
     conn.send JSON.stringify response if wrapper.ready()
   wrapper.ready = -> conn.readyState is 1
+  conn.on 'close', -> wrapper.emit 'close'
```

Comment:
Oh and, browserchannel emits this event on close, which is why we never saw memory  leaks with it:
https://github.com/josephg/node-browserchannel/blob/v1.2.0/lib/server.coffee#L845